ddtrace/internal/telemetry/writer.py

```diff
@@ -70,50 +88,68 @@ def send_event(self, request):
 
     def get_headers(self, request):
         # type: (Dict) -> Dict
-        """Get all telemetry api v1 request headers"""
+        """Get all telemetry api v2 request headers"""
         headers = self._headers.copy()
-        headers["DD-Telemetry-Debug-Enabled"] = request["debug"]
+        headers["DD-Telemetry-Debug-Enabled"] = str(request["debug"]).lower()
```

Comment:
The debug field should not be lower case ([PR](https://github.com/DataDog/dd-trace-py/commit/29826f959af174d44fd5af107be3631a0c170763))
```suggestion
        headers["DD-Telemetry-Debug-Enabled"] = request["debug"]
```